TESTS/psa/prot_internal_storage/COMPONENT_SPE/psa_setup.c

```diff
@@ -46,14 +46,14 @@ spm_partition_t g_partitions[3] = {
         .irq_mapper = NULL,
     },
     {
-        .partition_id = PSA_F_ID,
+        .partition_id = CRYPTO_SRV_ID,
```

Comment:
Do we want to change PSA_F_ID to CRYPTO_ID everywhere, even in non-crypto tests? Maybe the storage tests should use a storage partition and not a crypto partition?

Do the storage tests need Mbed Crypto? Do they need a Crypto partition?

Reply:
this is an auto-generated file which represents all the partitons defined in the system
specific for this test crypto and storage are the base partitions while its_reset is the additional user partition

and again this file will be compiled only for the secure side on dual-MCU(PSoC6) or v8-m systems only

Reply:
OK, so Crypto partition should always be available from the SPE.